torch/compiler/__init__.py

```diff
@@ -228,6 +229,25 @@ def disable(fn=None, recursive=True):
     return torch._dynamo.disable(fn, recursive)
 
 
+def set_phase(phase: str):
```

Comment:
`phase` seems like the wrong name here.  Forcing eager does not seem like a compilation phase.

There also uses of this that don't have anything to do with phases.